### PR TITLE
lets harvesters jaunt

### PR DESCRIPTION
**Pull request by ghost**
tbh for an endgame reward they're pretty goddamn unfun due to their lack of mobility and the fact that they take like 30 seconds to move one guy. 

#### Changelog

:cl:  
rscadd: Harvesters can jaunt like wraiths can, though without the refunds wraiths get for attacking someone.  
/:cl:

### CODE REVIEW

**Comment by Nirnael** [comment]
1. Left the default text on the PR, noob.

2. Buffs harvesters way too much making them OP so people can't run away.
👎

**Comment by ghost** [comment]
they're still as slow as normal constructs

**Comment by Nirnael** [comment]
But now you can't barricade those doors and roleplay a zombie apocalypse.

**Comment by ghost** [comment]
you're not supposed to ,you're supposed to run for the pods

**Comment by Nirnael** [comment]
Alright then. Now you can't barricade the pod doors and roleplay a zombie apocalypse.

**Comment by Nirnael** [comment]
I'm just joking adam, no problem.

**Comment by ghost** [comment]
i know there's not a problem i was just worried you were stupid enough to think there was one

**Comment by Nirnael** [comment]
I don't think you knew, good thinking on my part for telling you.

File: code/modules/mob/living/simple_animal/constructs.dm [hunk 1]
```diff
@@ -323,7 +323,8 @@
 	attacktext = "butchers"
 	attack_sound = 'sound/weapons/bladeslice.ogg'
 	construct_spells = list(/obj/effect/proc_holder/spell/aoe_turf/area_conversion,
-							/obj/effect/proc_holder/spell/targeted/forcewall/cult)
+							/obj/effect/proc_holder/spell/targeted/forcewall/cult,
+							obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift)
```
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [inline code/modules/mob/living/simple_animal/constructs.dm]: forgot a slash at the beginning

**Pull request merged**